phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexToolForNonTxGlobalIndexIT.java

```diff
@@ -0,0 +1,483 @@
+/*
```

Comment:
Thanks for moving the test cases that are specific to non transactional global indexes here. Good idea!